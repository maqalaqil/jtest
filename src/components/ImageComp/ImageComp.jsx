import React from 'react'
import image from '../../Untitled.png';

export default function ImageComp() {
    return (
        <div>
            <img src={image} alt="car" />
        </div>
    )
}
