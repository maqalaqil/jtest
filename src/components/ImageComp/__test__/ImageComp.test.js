import * as React from "react";
import * as renderer from "react-test-renderer";
import ImageComp from '../ImageComp'

it("this is working head", () => {
  const tree = renderer.create(<ImageComp />).toJSON();
  expect(tree).toMatchSnapshot();
});