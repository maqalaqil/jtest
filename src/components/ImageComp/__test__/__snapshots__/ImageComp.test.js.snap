// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is working head 1`] = `
<div>
  <img
    alt="car"
    src="Untitled.png"
  />
</div>
`;
